File: c-plus-plus/io/Load.cpp
#include <iostream>

static const auto NUMBER = 500000;

void run(int times) {
    int x = 0;

    for (auto i = 0; i <= times; i++) {
        x = i * i;
        std::cout << x << std::endl;
    }

    std::cout << x << std::endl;
}

int main() {
    run(NUMBER);
}

File: rust/io/Load.rs
const NUMBER: i64 = 500000;

fn run(times: i64) {
    let mut x = 0;
    for i in 0..times {
        x = i * i;
        println!("{}", x);
    }

    println!("{}", x);
}

fn main() {
    run(NUMBER);
}